macOS/fastlane/Fastfile

```diff
@@ -387,6 +467,30 @@ platform :mac do
     build_number
   end
 
+  # Gets the latest App Store version and build number for macOS
+  #
+  # @option [String] username (default: nil) Your DDG Apple ID
+  # @option [Boolean] live (default: true) Whether to get live version or TestFlight version
+  #
+  private_lane :fetch_appstore_version_info do |options|
+    app_store_build_number(
+      api_key: get_api_key,
+      username: get_username(options),
+      platform: 'osx',
+      live: options[:live] || true
+    )
+    
+    # The action sets these lane context variables:
+    # lane_context[SharedValues::LATEST_BUILD_NUMBER] - The build number
+    # lane_context[SharedValues::LATEST_VERSION] - The version number
+    
+    {
+      version: lane_context[SharedValues::LATEST_VERSION],
+      build_number: lane_context[SharedValues::LATEST_BUILD_NUMBER]
+    }
```

Comment:
This is not needed because you’re retrieving the values from lane_context anyway (lines 273-274 - and that is the preferred way btw 👍 )